OrbitQt/orbitmainwindow.cpp

```diff
@@ -105,19 +97,12 @@ OrbitMainWindow::OrbitMainWindow(QApplication* a_App,
   finalizing_capture_dialog->close();
 
   GOrbitApp->SetCaptureStopRequestedCallback([this, finalizing_capture_dialog] {
-    ui->actionToggle_Capture->setDisabled(true);
+    ui->actionToggle_Capture->setEnabled(false);
```

Comment:
I feel like this should also be captured by `UpdateCaptureStateDependentWidgets()`.

Reply:
I felt the same, but couldn't find a way to figure out if Orbit is in that state... I've added this as a local state in `OrbitMainWindow` now, let me know if there's a better way.

Reply:
I think that the way to do it would be to let `OrbitApp` expose `CaptureClient::state` instead of just `CaptureClient::IsCapturing`, which you use here through `OrbitApp::IsCapturing` and which only represents `state_ != State::kStopped`. "Stop requested" should then correspond to `State::kStopping`.

If the UI can be in all the states that correspond to the `CaptureClient::State`s, to me it makes sense to expose `CaptureClient::state` directly, otherwise with `CaptureClient::IsCapturing` (and `OrbitApp::IsCapturing`) we effectively lose information, that we then need to recover in different ways, such as what you did here with then adding `capture_stop_requested_`.

Reply:
Thanks for pointing this out, makes perfect sense. I added `GetCaptureState` to `OrbitApp`. I didn't remove `IsCapturing` at this point because it's used in more places and we have this pattern a lot through Orbit, but it could make sense to refactor these occurences in a later PR as well.

Reply:
Great, thanks!